x/superfluid/keeper/epoch_test.go

```diff
@@ -0,0 +1,100 @@
+package keeper_test
+
+import (
+	"errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	cltypes "github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/types"
+	gammtypes "github.com/osmosis-labs/osmosis/v15/x/gamm/types"
+	"github.com/osmosis-labs/osmosis/v15/x/superfluid/types"
+)
+
+func (suite *KeeperTestSuite) TestUpdateOsmoEquivalentMultipliers() {
+	testCases := []struct {
+		name               string
+		asset              types.SuperfluidAsset
+		expectedMultiplier sdk.Dec
+		removeStakingAsset bool
+		poolDoesNotExist   bool
+		expectedError      error
+	}{
+		{
+			name:               "update LP token Osmo equivalent successfully",
+			asset:              types.SuperfluidAsset{Denom: "gamm/pool/1", AssetType: types.SuperfluidAssetTypeLPShare},
```

Comment:
nit: this can be defined as a `defaultAsset` and reused

Reply:
https://github.com/osmosis-labs/osmosis/pull/4884/commits/3bda4def7126c4bb1b24c0c1b4d81332e125cd59